vtrace/platforms/linux.py

```diff
@@ -253,8 +253,8 @@ def setupMemFile(self, offset):
         """
         A utility to open (if necessary) and seek the memfile
         """
-        if self.memfd == None:
-            self.memfd = libc.open("/proc/%d/mem" % self.pid, O_RDWR | O_LARGEFILE, 0755)
+        if self.memfd is None:
+            self.memfd = libc.open("/proc/%d/mem" % self.pid, O_RDWR | O_LARGEFILE, 755)
```

Comment:
this is not the same either.  0755 is octal, equating to 493.

Reply:
Oh right. I use octal so little that I forgot about that. Thanks :+1: